detector/vuls2/vendor.go

```diff
@@ -40,6 +45,20 @@ func ignoreVulnerability(e ecosystemTypes.Ecosystem, v vulnerabilityTypes.Vulner
 			return true
 		}
 
+		return false
+	case ecosystemTypes.EcosystemTypeUbuntu:
+		if strings.HasPrefix(v.Content.Description, "** REJECT **") || strings.HasPrefix(v.Content.Description, "Rejected reason:") {
+			return true
+		}
+
+		if len(as) == 0 {
+			return false
+		}
+
+		if len(filterDistroAdvisories(e, as)) == 0 {
```

Comment:
It seems this condition will be never satisfied?

Reply:
If your data source is ubuntu-cve-tracker, then yes, there is no advisory.
However, if you choose a data source that can accurately add USNs, you may need to filter.

Reply:
Great!